Enterprise-CMCS/macpro-mdct-hcbs
# LTSS-5 Measure Overview Page

### Description
This PR adds the content for the LTSS-5 measure.
Only 4 questions for this one (2 are sub questions upon answering YES)

- "Were the reported measure results audited or validated?"
    - Upon answering Yes: "What is the name of the agency or entity that audited or validated the report?"
- "Did you deviate from the [reportYear] Technical Specifications?"
    - Upon answering Yes: "Please explain the deviation."
- Key difference in CMIT files for this one: 
    `measureSpecification:` `[MeasureSpecification.CMS],`
    `deliverySystem:` `[DeliverySystem.MLTSS],`
    - This is different than other measures who have two options for each
       `measureSpecification:` CMS & HEDIS
       `deliverySystem`: MLTSS & FSS

Notes from LTSS-1
1. Children textboxes of the radio button currently does not save any data. Address in a separate PR.
2. The quality measure table [Edit] buttons do not go anywhere yet. Address in a separate PR.

### Related ticket(s)
CMDCT-4147

---
### How to test

1. Sign into HCBS, any state user
2. Create a QM report
3. On the sidebar menu, select [Optional Measure Results]. 
   - The 6th row down will be LTSS-5, CMIT #1255.
4. Click [Edit] and you be taken to the LTSS-5 measure overview page.
5. Check that the questions matches what is written in the jira ticket: [CMDCT-4147](https://jiraent.cms.gov/browse/CMDCT-4147) - Less questions than other LTSS measures
6. You should see a page that looks like this
    ![LTSS-5](https://github.com/user-attachments/assets/fa6d9e65-424d-451d-81e8-f2c845217986)

### Important updates

---
### Author checklist


---
